docs/integrations/saas-cloud/dropbox.md

```diff
@@ -127,7 +127,7 @@ This section has instructions for installing the Dropbox App for Sumo Logic and
     1. **Data Source.** Select either of these options for the data source. 
       * Choose **Source Category**, and select a source category from the list. 
       * Choose **Enter a Custom Data Filter**, and enter a custom source category beginning with an underscore. Example: `_sourceCategory=MyCategory`. 
```

Comment:
If you indent these lines, it'll fix the roman numeral steps

```suggestion
       * Choose **Source Category**, and select a source category from the list. 
       * Choose **Enter a Custom Data Filter**, and enter a custom source category beginning with an underscore. Example: `_sourceCategory=MyCategory`. 
```

Reply:
indent right?